projects/current/projspec_single-file-catalog.md

```diff
@@ -247,7 +247,10 @@ or
 
 | Milestone        | Deadline   | Done    |
 |:-----------------|:----------:|:-------:|
-| 1. Lorem Ipsum.. | 2020-xx-xx | &#9744; |
+| 1. Move `esmcol-validator` into its own repo | 2020-01-21 | &#9744; |
+| 2. Update `esmcol-validator` and cut first release | 2020-01-21 | &#9744; |
+| 3. Incorporate `esmcol-validator` into intake-esm | 2020-01-22 | &#9744; |
+| 4. Support Single File Catalog in intake-esm | 2020-01-23 | &#9744; |
```

Comment:
@bonnland, does this deadline work for you? I am going to work on the rest of the items. I will do my best to meet their deadlines :)

Reply:
Yes, that should work.   We have the NCL Hackathon coming up, as you know, so I should have it done before that ideally.